cluster/ci/monitoring/mixins/prometheus_out/alertmanager.yaml

```diff
@@ -30,6 +36,14 @@ route:
   receiver: slack-notifications
   repeat_interval: 2h
   routes:
+  - continue: true
```

Comment:
`continue: true` would lead to a msg routed to the next receiver even though it matches already the current one.
Is that what we want?